hack/lib/Dockerfile-origin-release

```diff
@@ -0,0 +1,9 @@
+FROM registry.svc.ci.openshift.org/openshift/origin-release:v4.0 as origin-release
+
+FROM fedora:29 as jq
```

Comment:
Fedora and not centos?

Reply:
What does `as jq` line does here?

Reply:
not sure if `jq` is avaiable in centos:7, however I would probably just use some minimal jq (scratch image exists?).

Reply:
or will just instead of yum install curl it directly on centos:7 so I don't have to pull the fedora layers ;-)

Reply:
There's none by default, you need to install `epel-release` and only then `jq`. Or just `wget` it.